doc/api/fs.md

```diff
@@ -2764,6 +2764,26 @@ The callback is given the three arguments, `(err, bytesRead, buffer)`.
 If this method is invoked as its [`util.promisify()`][]ed version, it returns
 a `Promise` for an `Object` with `bytesRead` and `buffer` properties.
 
+## `fs.read(fd, options, callback)`
+<!-- YAML
+added: v0.0.2
```

Comment:
```suggestion
added: REPLACEME
```

Reply:
given that this is a new section separate from the other `fs.read()` mention, it should have it's own yaml history.

Reply:
yea,  copy/paste is hard :)